src/test/java/com/coravy/hudson/plugins/github/GitHubRepositoryNameTest.java

```diff
@@ -23,7 +23,7 @@
 @RunWith(DataProviderRunner.class)
 public class GitHubRepositoryNameTest {
 
-    public static final String FULL_REPO_NAME = "jenkinsci/jenkins";
+    public static final String FULL_REPO_NAME = "jenkins/jenkins";
```

Comment:
What exactly is the point of the change here... https://github.com/jenkins/jenkins is a 404 for me

<img width="976" alt="screen shot 2017-07-31 at 10 51 06" src="https://user-images.githubusercontent.com/209336/28772459-35726fba-75de-11e7-8dc6-bb105283b1b1.png">